### DISCUSSION TITLE
More Color

### DISCUSSION THREAD

**Discussion by kaesekarl** [outline/outline#2003] [Ideas]
What about more Coloroptions for Text? I personally find it very restricting to only have one highlighter available. I would appreciate a few more colors for the highlighter and also for the text itself. I Personally would to include a red word which is crucial etc.

Maybe include that also in the slash-menu?

**Comment by greg-thrive** [comment]
100% agree @kaesekarl   I like green, and there's no green.

**Comment by Mag1cByt3s** [comment]
Is there any update on this?

I finally set up outline after a few hours and now I want to
change the syntax highlighting theme because I don't like it.

I don't see any option to change the theme so I looked in the wiki and here.

Is there any solution to this?

**Comment by WarlockFish** [comment]
This function is useful. Are there any plans to add it？  @tommoor

**Comment by wouter-deen** [comment]
Honestly this is just an essential feature. Why can't we change the color of text?

**Comment by msklywenn** [comment]
I'd like this too. But I guess it's a problem because Markdown doesn't support color and Outline wishes to keep compatibility with it?

**Comment by Limezy** [comment]
As per my understanding Outline is slowly getting away from Markdown, but new features such as these color options are still blocked until they move the backend to be Json based instead of markdown based

**Comment by EduNetEurope** [comment]
It would be amazing to have both the ability to select custom colours for texts bits and to select custom colours for the highlighter.

Here are two related requests:
- https://github.com/outline/outline/discussions/5101
- https://github.com/outline/outline/discussions/4745

**Comment by Cristian-PGD** [comment]
I guess is still not resolved

**Comment by msklywenn** [comment]
There are five colors now. To me, this is fixed.

**Reply by wasdennnoch** [reply]
There are highlight colors now but not text colors, which the original post also asked for.

**Comment by coffeeuser26** [comment]
Text colors would be great!
